beacon-chain/rpc/validator_server_test.go

```diff
@@ -182,11 +183,22 @@ func TestNextEpochCommitteeAssignment_OK(t *testing.T) {
 	if err := db.UpdateChainHead(genesis, state); err != nil {
 		t.Fatalf("Could not save genesis state: %v", err)
 	}
-
-	for i := 0; i < int(params.BeaconConfig().DepositsForChainStart); i++ {
+	var wg sync.WaitGroup
+	numOfValidators := int(params.BeaconConfig().DepositsForChainStart)
+	errs := make(chan error, numOfValidators)
+	for i := 0; i < numOfValidators; i++ {
 		pubKeyBuf := make([]byte, binary.MaxVarintLen64)
 		n := binary.PutUvarint(pubKeyBuf, uint64(i))
-		if err := db.SaveValidatorIndex(pubKeyBuf[:n], i); err != nil {
+		wg.Add(1)
+		go func(itemID string) {
```

Comment:
so you dont use `itemID` here, so not sure what is the use case. Why use strings when indexes are `int` ?

Reply:
my mistake

Reply:
fixed it